File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.cpp
class Solution {
public:
    bool checkIfPangram(string sentence) {
        bool s[26] = {false};

        for(int i=0;i<sentence.length();i++){
            s[sentence[i] - 'a'] = true;

        }

        for(int i=0;i<26;i++){
            if(s[i] == false){
                return false;
            }
        }
        return true;
        
    }
};

File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.java
class Solution {
    public boolean checkIfPangram(String sentence) {
        boolean[] s = new boolean[26];

        for(int i=0;i<sentence.length();i++){
            s[sentence.charAt(i) - 'a'] = true;
        }
        for(int j=0;j<26;j++){
            if(s[j] == false){
                return false;
            }
        }

        return true;
        
    }
}